neuraxle_tensorflow/tensorflow_v2.py

```diff
@@ -54,13 +65,13 @@ def setup(self) -> BaseStep:
         if self.is_initialized:
             return self
 
-        self.optimizer = self.create_optimizer()
-        self.model = self.create_model()
+        self.optimizer = self.create_optimizer(self)
+        self.model = self.create_model(self)
 
         self.checkpoint = tf.train.Checkpoint(step=tf.Variable(1), optimizer=self.optimizer, net=self.model)
         self.checkpoint_manager = tf.train.CheckpointManager(
             self.checkpoint,
-            self.checkpoint_folder,
+            self.tf_model_checkpoint_folder,
```

Comment:
note that I rarily configure checkpoints with my TensorFlow models, although this would be good for me to do early stopping sometimes. 

I would ask here that if the path to the checkpoint is not defined, that there be just no checkpoint. It looks like here we always checkpoint something, if I'm not wrong, but we shouldn't force users to use checkpoints.

Reply:
no we don't always checkpoint. the checkpoints are only used with the saver.

Reply:
it is already facultative in the constructor of the step. closing this

Reply:
ok